src/OrchardCore.Modules/OrchardCore.Lists/Services/IContainerService.cs

```diff
@@ -10,7 +12,7 @@ public interface IContainerService
         /// <summary>
         /// Query contained items by page either order by the created utc or order value.
         /// </summary>
-        Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, bool publishedOnly);
+        Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, ContentsStatus contentsStatus, ContainedItemOptions listPartFilter);
```

Comment:
A `ContentsStatus` benne van a `ContainedItemOptions`-ben is, az előbbi paraméterre nincsen szükség. Már csak azért sem, mert nem is használja semmi. A `listPartFilter` pedig outdated.

Reply:
Továbbra is.

Reply:
Akkor ha jól értem 5 helyett elég 4 paraméter, és a ContentsStatus-ra nincs is szükség. Illetve a DisplayDriver-ben a metódushíváskor sem lesz rá szükség, simán kitörölhetem. És akkor az ottani conversation (az e fölötti) semmis, mert semmilyen ContentsStatus átadására nincs szükség.

Reply:
A driverben valahogy át kell majd adni a statust, ahogy régen is, csak a `ContainedItemOptions`-ben, hacsak nem az ott korábban átadott a default `null` esetén.

Reply:
Most töröltem a ContentsStatus-os paramétert. 
Alsó metódusban a ContainedItemOptions -be bele van másolva a viewmodel-ből a status értéke is, és így van paraméterként átadva a példány, tehát elvileg jónak kell lennie